io/www/src/fragment/settings.js

```diff
@@ -1,3 +1,34 @@
+const COLLECTION_MODEL_ID =
+    'L2NvbmYvbWFzL3NldHRpbmdzL2RhbS9jZm0vbW9kZWxzL2NvbGxlY3Rpb24';
+
+function applyCollectionSettings(context) {
+    context.body.placeholders = {
+        searchText: '{{coll-search-text}}',
+        filtersText: '{{coll-filters-text}}',
+        sortText: '{{coll-sort-text}}',
+        popularityText: '{{coll-popularity-text}}',
+        alphabeticallyText: '{{coll-alphabetically-text}}',
+        noResultsText: '{{coll-no-results-text}}',
+        plansSidenavTitle: '{{coll-plans-sidenav-title}}',
+        resultText: '{{coll-result-text}}',
+        resultsText: '{{coll-results-text}}',
+        resultMobileText: '{{coll-result-mobile-text}}',
+        resultsMobileText: '{{coll-results-mobile-text}}',
+        searchResultText: '{{coll-search-result-text}}',
+        searchResultsText: '{{coll-search-results-text}}',
+        searchResultMobileText: '{{coll-search-result-mobile-text}}',
+        searchResultsMobileText: '{{coll-search-results-mobile-text}}',
+        noSearchResultsText: '{{coll-no-search-results-text}}',
+        showMoreText: '{{coll-show-more-text}}',
+    };
+    context.dictionary = {
```

Comment:
can `dictionary` be baked in frontend code? Are they going to be ever updated via authoring somehow?

Reply:
i don't think they should, this is dev to dev kind of thing: tech {{ }} placeholder gives tech <span> placeholder that our own code uses.

Reply:
above, are you reference to `dictionary` as tech placeholders?

Reply:
yes, because those entries are in dictionnary _before_ we fetch entries from user (odin)